addons/packages/calico/3.22.1/bundle/config/overlay/calico_overlay.yaml

```diff
@@ -0,0 +1,150 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("/values.star", "values")
+#@ load("@ytt:json", "json")
+
+#@ def startswith(index, left, right):
+#@  return left.startswith(right)
+#@ end
+
+#@overlay/match by=overlay.subset({"kind":"Deployment","metadata":{"name": "calico-kube-controllers"}})
+---
+kind: Deployment
+metadata:
+  name: calico-kube-controllers
+  #@overlay/match missing_ok=True
+  annotations:
+    kapp.k14s.io/disable-default-label-scoping-rules: ""
+    kapp.k14s.io/update-strategy: fallback-on-replace
+
+#@overlay/match by=overlay.subset({"kind":"DaemonSet", "metadata": {"name": "calico-node"}})
+---
+kind: DaemonSet
+metadata:
+  #@overlay/match missing_ok=True
+  annotations:
+    kapp.k14s.io/disable-default-label-scoping-rules: ""
+    kapp.k14s.io/update-strategy: fallback-on-replace
```

Comment:
Hi @vijaykatam , in this v3.22.1, I followed your suggestion to change the update strategy to `fallback-on-replace` for both calico daemonset and calico deployment, feel free to take a look and leave any comments, thanks.

Reply:
@12345lcr Sorry, just re-read your comment above. You can use the `push-package` make task in the TCE repo, just use a tag like `3.22.1-2`. We've enabled immutable tags, so we can no longer re-use tags.